tests/test_api_policy.py

```diff
@@ -268,6 +268,65 @@ def test_02_set_policy_conditions(self):
             res = self.app.full_dispatch_request()
             self.assertEqual(res.status_code, 200)
 
+    def test_03_set_adminuser_policy(self):
+        # Set a policy for the user testadmin
+        with self.app.test_request_context('/policy/pol1adminuser',
+                                           method='POST',
+                                           data={
+                                               "action": "{0!s}, {1!s}".format(ACTION.POLICYDELETE,
+                                               ACTION.POLICYREAD),
+                                               "scope": SCOPE.ADMIN,
+                                               "realm": "",
+                                               "adminuser": "testadmin"},
+                                           headers={
+                                               'Authorization': self.at}):
+            res = self.app.full_dispatch_request()
+            self.assertTrue(res.status_code == 200, res)
+            data = res.json
+            result = data.get("result")
+            self.assertTrue("setPolicy pol1adminuser" in result.get("value"),
+                            result.get("value"))
+
+        # Get the policies
+        with self.app.test_request_context('/policy/',
+                                           method='GET',
+                                           headers={
+                                               'Authorization': self.at}):
+            res = self.app.full_dispatch_request()
+            self.assertTrue(res.status_code == 200, res)
+            data = res.json
+            result = data.get("result")
+            policies = result.get("value")
+            # We have two polices
+            self.assertEqual(2, len(policies))
```

Comment:
Why are there two policies? We should really try to make test-runs more independent. Is the policy from the previous test still needed? Then maybe we should add/remove it in the setUp/tearDown functions.

Reply:
OK, will check.

Reply:
I actually see this a bit more relaxed. This is why we can have test classes, so that several tests could build on each other. However, done in 0798f7a598ec6a111c0e64955240d01cd6c7780f